pkg/client/monitoring/v1alpha1/types.go

```diff
@@ -125,6 +127,16 @@ type PrometheusStatus struct {
 	UnavailableReplicas int32 `json:"unavailableReplicas"`
 }
 
+// LogSpec defines the configured log messages from the Prometheus logger.
+type LogSpec struct {
```

Comment:
Turns out this was a good call :slightly_smiling_face: seems like the semantics are going to change quite a bit, I'd suggest we hold off until we fully support 2.0 (and 2.0 has a stable release) https://github.com/prometheus/prometheus/pull/3061